api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -2236,42 +2223,69 @@ public void testCloneWorkspaceIncludeUserRoles() {
                         .put("canCompute", true)
                         .put("canShare", true)));
 
+    // cloning/duplication is not atomic. When the workspace is first created, it will only have
+    // creator=OWNER access, like a newly-created workspace.  We add other permissions later.
+
+    RawlsWorkspaceACL clonedAclBeforeUpdate =
+        createWorkspaceACL(
+            new JSONObject()
+                .put(
+                    "cloner@gmail.com",
+                    new JSONObject()
+                        .put("accessLevel", "OWNER")
+                        .put("canCompute", true)
+                        .put("canShare", true)));
+
+    when(fireCloudService.getWorkspaceAclAsService(
+            originalWorkspace.getNamespace(), originalWorkspace.getName()))
+        .thenReturn(originalAcl);
     when(fireCloudService.getWorkspaceAclAsService("cloned-ns", "cloned"))
-        .thenReturn(workspaceAclFromCloned);
-    when(fireCloudService.getWorkspaceAclAsService(workspace.getNamespace(), workspace.getName()))
-        .thenReturn(workspaceAclFromOriginal);
+        .thenReturn(clonedAclBeforeUpdate);
 
-    currentUser = cloner;
+    // cloner is now OWNER, and the "published" group has NO_ACCESS
+    List<RawlsWorkspaceACLUpdate> expectedCollaboratorsAfterUpdate =
+        convertUserRolesToUpdateAclRequestList(
+            Set.of(
+                new UserRole().email(cloner.getUsername()).role(WorkspaceAccessLevel.OWNER),
+                new UserRole().email(LOGGED_IN_USER_EMAIL).role(WorkspaceAccessLevel.OWNER),
+                new UserRole().email(reader.getUsername()).role(WorkspaceAccessLevel.READER),
+                new UserRole().email(writer.getUsername()).role(WorkspaceAccessLevel.WRITER),
+                new UserRole()
+                    .email(workspaceService.getPublishedWorkspacesGroupEmail())
+                    .role(WorkspaceAccessLevel.NO_ACCESS)));
 
-    Workspace modWorkspace =
```

Comment:
This was only used to construct a `CloneWorkspaceRequest` so it is now `cloneRequest` below